avl_tree/AvlTree.java

```diff
@@ -0,0 +1,265 @@
+import java.util.Scanner;
+import java.util.NoSuchElementException;
+import java.util.InputMismatchException;
+import java.util.Random;
+
+public class AvlTree {
+
+	public static void main(String[] args) {
+		AVL avl = new AVL();
+		Random rand = new Random();
+		int inp = 10000;
+		for (int i = 0; i < inp; i++) {
+			avl.root = avl.insert(avl.root, rand.nextInt(10000));
+		}
+		avl.delNode(avl.root, 1);
+		avl.prntIn(avl.root);
+	}
+}
+
+class AVL {
+	public NodeAVL root = null;

```

Comment:
Don't directly assign it null, make a constructor for this purpose.
Also you can make a parameterized constructor in case you want an existing node to be the root.